src/main/java/org/mskcc/cbio/oncokb/web/rest/UsageAnalysisController.java

```diff
@@ -0,0 +1,140 @@
+package org.mskcc.cbio.oncokb.web.rest;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.PostConstruct;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.S3Object;
+import com.amazonaws.services.s3.model.S3ObjectInputStream;
+import com.google.gson.Gson;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.mskcc.cbio.oncokb.config.application.ApplicationProperties;
+import org.mskcc.cbio.oncokb.web.rest.vm.usageAnalysis.UsageSummary;
+import org.mskcc.cbio.oncokb.web.rest.vm.usageAnalysis.UserOverviewUsage;
+import org.mskcc.cbio.oncokb.web.rest.vm.usageAnalysis.UserUsage;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/api")
+public class UsageAnalysisController {
+
+    @Autowired
+    private ApplicationProperties applicationProperties;
+
+    private String s3AccessKey = "";
+    private String s3SecretKey = "";
+    private String s3Region = "";
+
+    final String ONCOKB_BUCKET = "oncokb";
+    final String USERS_USAGE_SUMMARY_FILE = "usage-analysis/userSummary.json";
+    final String RESOURCES_USAGE_SUMMARY_FILE = "usage-analysis/resourceSummary.json";
+
+    @PostConstruct
+    private void init() {
+        s3AccessKey = applicationProperties.getAws().getS3AccessKey();
+        s3SecretKey = applicationProperties.getAws().getS3SecretKey();
+        s3Region = applicationProperties.getAws().getS3Region();
+    }
+
+    private JSONObject requestData(String bucket, String file)
+            throws UnsupportedEncodingException, IOException, ParseException {
+        AWSCredentials credentials = new BasicAWSCredentials(s3AccessKey, s3SecretKey);
+        AmazonS3 s3client = AmazonS3ClientBuilder.standard()
+                .withCredentials(new AWSStaticCredentialsProvider(credentials)).withRegion(s3Region).build();
+
+        S3Object s3object = s3client.getObject(bucket, file);
```

Comment:
Let's put the bucket here. This particular method is only for oncokb bucket anyway